google/cloud/bigtable/idempotent_mutation_policy.cc

```diff
@@ -44,7 +44,7 @@ std::unique_ptr<IdempotentMutationPolicy> AlwaysRetryMutationPolicy::clone()
 }
 
 bool AlwaysRetryMutationPolicy::is_idempotent(
-    google::bigtable::v2::Mutation const& m) {
+    google::bigtable::v2::Mutation const& /*m*/) {
```

Comment:
FTR, I'm happy with (and slightly prefer) omitting unused parameter names. I think the clang-tidy check should be happy w/ the name omitted. The Google style guide allows omission. So keeping the unused parameter names commented out may be fine, but it would be a local style convention of ours.

Happy to approve this change as-is, because either way it's no big deal.

Reply:
I agree with you, and even if I did not, I see no benefit to have our own standard for this.  Unfortunately, I get this when I remove the parameter altogether:

```
/v/google/cloud/bigtable/idempotent_mutation_policy.cc:47:42: error: all parameters should be named in a function [readability-named-parameter,-warnings-as-errors]
    google::bigtable::v2::Mutation const&) {
```

So this check (`misc-*`) is happy with the parameter omitted, but `readability-*` is not :confused: 

I can also disable the `readability-named-parameter` check and get this to work, it is pretty easy.  Thoughts?

Reply:
I think that's a bug in the readability-named-parameter clang-tidy check. The docs for that check say it checks for conformance to [Google's style guide](https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions), but the style guide specifically says "A parameter name may be omitted only if the parameter is not used in the function's definition."

I suggest we just stick `-readability-named-parameter` to not use that check. We don't use it internally, fwiw.

If I were less lazy, I might file a bug against this check.